sale_prequotation/sale_prequotation.py

```diff
@@ -869,6 +873,78 @@ def action_sale(self, cr, uid, ids, format_quote, context=None):
         result['domain'] = "[('id','in', [" + ','.join(map(str, quote_ids)) + "])]"
         return result
 
+    def action_new_pq_version(self, cr, uid, ids, context=None):
+        pq = self.pool.get('sale.prequotation')
+        pq_product = self.pool.get('sale.prequotation.product')
+        pq_labour = self.pool.get('sale.prequotation.labour')
+        margin = self.pool.get('sale.profit.margin')
+        for prequote in self.browse(cr, uid, ids, context=context):
+            vals = {
+                'name': prequote.name + '-' + NUMBER_TO_CHAR[prequote.pq_version],
+                'project_name': prequote.project_name,
+                'partner_id': prequote.partner_id.id,
+                'date_pq': prequote.date_pq,
+                'cal_date': prequote.cal_date,
+                'user_id': uid,
+                'overwrite_uom': prequote.overwrite_uom,
+                'overall_product_type': prequote.overall_product_type,
+                'mat_sale_id': prequote.mat_sale_id.id,
+                'labour_sale_id': prequote.labour_sale_id.id,
+                'mat_sale_no': prequote.mat_sale_no,
+                'labour_sale_no': prequote.labour_sale_no,
+                'doc_version': prequote.doc_version,
+                'pq_version': prequote.pq_version,
+                'pq_id': prequote.id,
+                'customer_commission': prequote.customer_commission,
+                'percent_unforeseen': prequote.percent_unforeseen,
+            }
+            pq_older = pq.create(cr, uid, vals, context=context)
```

Comment:
```suggestion
            pq_new = pq.create(cr, uid, vals, context=context)
```
Should be use `pq_new` and `pq_origin` are good?